src/components/home/FAQ.tsx

```diff
@@ -10,12 +10,12 @@ export default function FAQ() {
               <dl className="w-full md:w-2/3">
                   <dt className="mb-4">
                       <h3 className="text-xl font-bold leading-snug tracking-tight mb-1">
-                          What motivated you all to make this app?
+                          What motivated you to make this app?
                       </h3>
                   </dt>
                   <dd className="mb-16">
                       <p className="text-gray-600">
-                          We are a trio of early-career developers. While searching for ideas to hone our skills, we came across the challenge of creating an app that faciliated learning languages through reading. Although a number of apps exist in either open source or paid form, none of them met the full spectrum of needs based on the checklist of our language-learning enthusiast team member.
+                          We are a trio of software engineers. While searching for ideas to hone our skills, we came across the challenge of creating an app that faciliated learning languages through reading. Although a number of apps exist in either open source or paid form, none of them met the full spectrum of needs based on the checklist of our language-learning enthusiast team member.
```

Comment:
Maybe even shorter like this:
"We are a trio of software engineers who came across the challenge of creating an app that facilitated learning languages through reading. Although a number of apps exist in either open source or paid form, none of them met the full spectrum of needs based on the checklist of our language-learning enthusiast team member."